deploy/scripts/003_deploy_actions.ts

```diff
@@ -0,0 +1,63 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { ethers, getChainId } from "hardhat";
+import { addToWhitelist, getLibraries } from "../utils";
+import { Contract } from "ethers";
+import dotenv from "dotenv";
+
+const func: DeployFunction = async function(hre: HardhatRuntimeEnvironment) {
+  dotenv.config({ path: (await getChainId()) == "31337" ? ".test.env" : ".env" });
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const { TokenLibAddr } = await getLibraries();
+
+  const whitelistService = await ethers.getContract("WhitelistService");
+  const actWlHash = await whitelistService.getWhitelistHash(deployer, "actions");
+  console.log("actWlHash", actWlHash);
+  try {
+    await whitelistService.createWhitelist("actions");
+  } catch {
+    // loose test for "that that whitelist was already created"
+  }
+
+  await deploySwapUniSingleAction(deploy, deployer, whitelistService, actWlHash, TokenLibAddr);
+  // TODO: deploy all the other actions
+
+  if ((await whitelistService.getWhitelistOwner(actWlHash)) == deployer) {
+    await whitelistService.transferWhitelistOwnership(actWlHash, process.env.PLATFORM_MULTI_SIG_ADDR);
+  } // else was already transferred
+};
+
+async function deploySwapUniSingleAction(
+  deploy: any,
+  deployer: string,
+  whitelistService: Contract,
+  actWlHash: any,
+  TokenLibAddr: string
+) {
+  let uniswapAddr;
+  let weth9Addr;
+
+  // TODO: utils to change the following to vars, depending on chainID
```

Comment:
yeah we should collect all this test vs prod branches  in 1 place instead of scattering them all over. that way, we will know what differences exists between test and prod deployments

Reply:
I don't follow